app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java

```diff
@@ -68,27 +68,27 @@ protected void onCreate(Bundle savedInstanceState) {
         //Creating the single select dropdown menu for the trap outcomes
         Spinner trapSpinner=findViewById(R.id.spinnerTrap);
         //accessing the array in strings.xml
-        ArrayAdapter<CharSequence> trapAdapter= ArrayAdapter.createFromResource(this,R.array.trap_outcomes_array, android.R.layout.simple_spinner_item);
-        trapAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
-        trapSpinner.setAdapter(trapAdapter);
+//        ArrayAdapter<CharSequence> trapAdapter= ArrayAdapter.createFromResource(this,R.array.trap_outcomes_array, android.R.layout.simple_spinner_item);
+//        trapAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
```

Comment:
you should remove these lines.

Reply:
I didn't want to mess with Ella's code because she will probably still use it.